Libraries/Utilities/useWindowDimensions.js

```diff
@@ -16,7 +16,7 @@ import * as React from 'react';
 
 export default function useWindowDimensions(): DisplayMetrics {
   const dims = Dimensions.get('window'); // always read the latest value
-  const forceUpdate = React.useState()[1];
+  const forceUpdate = React.useState()[1].bind(this, v => !v);
```

Comment:
Could bind `null` instead of `this`? Also probably want to pass `false` explicitly as the initial value for cleaner typing.